Frends.AmazonS3.ListObjects/Frends.AmazonS3.ListObjects.Test/UnitTests.cs

```diff
@@ -0,0 +1,126 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using Frends.AmazonS3.ListObjects.Definitions;
+
+namespace Frends.AmazonS3.ListObjects.Test
+{
+    [TestClass]
+    public class ListObjectsTest
+    {
+        private readonly string _accessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_AccessKey");
+        private readonly string _secretAccessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_SecretAccessKey");
+        private readonly string _bucketName = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_BucketName");
+
+        Source _source;
+        Options _options;
+
+
+        /// <summary>
+        /// List all objects.
+        /// </summary>
+        [TestMethod]
+        public void ListAll()
+        {
+            _source = new Source
+            {
+                AwsAccessKeyId = _accessKey,
+                AwsSecretAccessKey = _secretAccessKey,
+                BucketName = _bucketName,
+                Region = Regions.EuCentral1
+            };
+
+            _options = new Options
+            {
+                ContinuationToken = null,
+                Delimiter = null,
+                MaxKeys = 100,
+                Prefix = null,
+                StartAfter = null
+            };
+
+            var result = AmazonS3.ListObjects(_source, _options, default);
+            Assert.IsNotNull(result.Result.ObjectList);
+        }
+
+        /// <summary>
+        /// StartAfter.
+        /// </summary>
+        [TestMethod]
+        public void StartAfter()
+        {
+            _source = new Source
+            {
+                AwsAccessKeyId = _accessKey,
+                AwsSecretAccessKey = _secretAccessKey,
+                BucketName = _bucketName,
+                Region = Regions.EuCentral1
+            };
+
+            _options = new Options
+            {
+                ContinuationToken = null,
+                Delimiter = null,
+                MaxKeys = 100,
+                Prefix = null,
+                StartAfter = "2022-04-22T00:16:40+02:00"
+            };
+
+            var result = AmazonS3.ListObjects(_source, _options, default);
+            Assert.IsNotNull(result.Result.ObjectList);
+        }
+
+        /// <summary>
+        /// Prefix.
+        /// </summary>
+        [TestMethod]
+        public void Prefix()
```

Comment:
This name should also contain `Test`. Also, the name should describe the test a little bit better.